addons/nuxeo-drive-server/nuxeo-drive-core/src/main/resources/OSGI-INF/nuxeodrive-bulk-action-contrib.xml

```diff
@@ -9,7 +9,8 @@
   <extension target="org.nuxeo.runtime.stream.service"
     point="streamProcessor">
     <streamProcessor name="driveFireGroupUpdatedEvent" class="org.nuxeo.drive.action.FireGroupUpdatedEventAction"
-      logConfig="bulk">
+      logConfig="bulk" defaultConcurrency="${nuxeo.bulk.action.driveFireGroupUpdatedEvent.defaultConcurrency:=2}"
+      defaultPartitions="${nuxeo.bulk.action.driveFireGroupUpdatedEvent.defaultPartitions:=4}">
```

Comment:
Any reason why 4?

Reply:
this is the same default as before, just set explicitely